lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -289,7 +289,13 @@ protected function _getPortableSequenceDefinition($sequence)
             $sequenceName = $sequence['relname'];
         }
 
-        $data = $this->_conn->fetchAll('SELECT min_value, increment_by FROM ' . $this->_platform->quoteIdentifier($sequenceName));
+        $version = (float) $this->_conn->getWrappedConnection()->getServerVersion();
```

Comment:
This is breaking the test suite, can you please check that?

Reply:
Also [Postgres 10 yet is not supported in Travis](https://docs.travis-ci.com/user/database-setup/#Using-a-different-PostgreSQL-Version), so there's no way to 100% guarantee the compatibility at this moment.